packages/mobile/src/screens/sign-up-screen/screens/PickHandleScreen.tsx

```diff
@@ -27,10 +35,51 @@ type PickHandleValues = {
   handle: string
 }
 
+type SocialMediaSectionProps = {
+  onCompleteSocialMediaLogin: (info: {
+    requiresReview: boolean
+    handle: string
+    platform: 'twitter' | 'instagram' | 'tiktok'
+  }) => void
+}
+
+const SocialMediaSection = ({
+  onCompleteSocialMediaLogin
+}: SocialMediaSectionProps) => {
+  const { spacing } = useTheme()
+  return (
+    <Paper direction='column' backgroundColor='surface2' p='l' gap='l'>
+      <Flex direction='column' gap='s'>
+        <Text variant='title' size='m'>
+          {messages.claimHandleHeaderPrefix}{' '}
+          <Text color='accent'>@{messages.handle}</Text>
+          <IconVerified size='s' style={{ marginLeft: spacing.xs }} />
+        </Text>
+        <Text variant='body' size='m'>
+          {messages.claimHandleDescription}
+        </Text>
+      </Flex>
+      <SocialMediaLoginOptions
+        onCompleteSocialMediaLogin={onCompleteSocialMediaLogin}
+      />
+      <Text variant='body' size='m'>
+        {messages.claimHandleHeadsUp}
+      </Text>
+    </Paper>
+  )
+}
+
 export const PickHandleScreen = () => {
   const navigation = useNavigation<SignUpScreenParamList>()
   const dispatch = useDispatch()
 
+  const audiusQueryContext = useAudiusQueryContext()
+  const validationSchema = useMemo(() => {
```

Comment:
if you want to define this outside the component, can prob just import audiusQueryContext directly from the context file?